Azure.PowerShell.Netcore.sln

```diff
@@ -1,50 +1,52 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 15
-VisualStudioVersion = 15.0.27019.1
+VisualStudioVersion = 15.0.27130.2010
```

Comment:
Why is this file changed?  Did we need to add in the strategies project?

Reply:
The netcore build failed, so I had to do something with it, but maybe it's not needed to change the solution file. I will try to revert this particular part and see.